pilot/pkg/model/jwks_resolver.go

```diff
@@ -380,13 +387,17 @@ func (r *JwksResolver) getRemoteContentWithRetry(uri string, retry int) ([]byte,
 func (r *JwksResolver) refresher() {
 	// Wake up once in a while and refresh stale items.
 	r.refreshTicker = time.NewTicker(r.refreshInterval)
-	lastHasError := false
+	lastHasError := tickerCounter - 1
+	if r.refreshIntervalOnFailure == time.Second {
```

Comment:
This looks like horrible hack to compare with a constant `time.Second` ...

Reply:
cool, will change this!!